### PR TITLE
[4.x] Ability to see which masters are paused and only show paused if everything is paused

### PR DESCRIPTION
**Pull request by glamorous**
This pull request does two things:

- Show on a every "master" if it's paused or running
- If horizon is active but at least 1 master is running and 1 is paused, you see this in the dashboard.

A couple of images to show what has been changed.

status_workers
inactive-not-changed
active_unchanged
paused_unchanged
active_changed

_Little question: I wasn't sure if I should update the javascript files by running `npm run production`. If necessary, I can make changes._

Closes https://github.com/laravel/horizon/issues/927

### CODE REVIEW

File: tests/Controller/DashboardStatsControllerTest.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ public function test_paused_status_is_reflected_if_all_master_supervisors_are_pa
         $masters = Mockery::mock(MasterSupervisorRepository::class);
         $masters->shouldReceive('all')->andReturn([
             (object) [
-                'status' => 'running',
+                'status' => 'paused',
```
Comment (driesvints) [inline tests/Controller/DashboardStatsControllerTest.php:77]: Why was this changed?

Comment (glamorous) [reply tests/Controller/DashboardStatsControllerTest.php:77]: Because now it only will show "paused" if all master supervisorrs are paused. Changed in behaviour because it's now unclear if you look at the dashboard. Now it shows "paused" while only one master supervisor is paused and everything is running. Therefore this pull request to show users (see images) if not all master supervisors are paused or not. Test was failing because he expected to be "paused" if only one is "paused" despite the wrong title that was indicating all master supervisors are paused.

**Comment by taylorotwell** [comment]
This was sent to 4.x branch. The current release of Horizon is 5.x. Please submit to that branch.

**Comment by glamorous** [comment]
Opened a new PR: https://github.com/laravel/horizon/pull/929
